oxide-vpc/tests/integration_tests.rs

```diff
@@ -485,72 +488,100 @@ const UFT_LIMIT: Option<NonZeroU32> = NonZeroU32::new(16);
-        gw_ip: "172.30.0.1".parse().unwrap(),
+        gateway_mac: MacAddr::from([0xA8, 0x40, 0x25, 0xFF, 0x77, 0x77]),
         vni: Vni::new(1287581u32).unwrap(),
         // Site 0xF7, Rack 1, Sled 1, Interface 1
         phys_ip: Ipv6Addr::from([
             0xFD00, 0x0000, 0x00F7, 0x0101, 0x0000, 0x0000, 0x0000, 0x0001,
         ]),
-        bsvc_addr: PhysNet {
-            ether: MacAddr::from([0xA8, 0x40, 0x25, 0x77, 0x77, 0x77]),
+        boundary_services: BoundaryServices {
+            mac: MacAddr::from([0xA8, 0x40, 0x25, 0x77, 0x77, 0x77]),
             ip: Ipv6Addr::from([
                 0xFD, 0x00, 0x99, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
                 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
             ]),
             vni: Vni::new(99u32).unwrap(),
         },
         proxy_arp_enable: false,
-        phys_gw_mac: None,
+        phys_gw_mac: Some(MacAddr::from([0x78, 0x23, 0xae, 0x5d, 0x4f, 0x0d])),
     }
 }
 
 fn g2_cfg() -> VpcCfg {
+    let ip_cfg = IpCfg::DualStack {
+        ipv4: Ipv4Cfg {
+            vpc_subnet: "172.30.0.0/22".parse().unwrap(),
+            private_ip: "172.30.0.6".parse().unwrap(),
+            gateway_ip: "172.30.0.1".parse().unwrap(),
+            snat_cfg: Some(SNat4Cfg {
+                // NOTE: This is not a routable IP, but remember that a
+                // "public IP" for an Oxide guest could either be a
+                // public, routable IP or simply an IP on their wider LAN
+                // which the oxide Rack is simply a part of.
+                public_ip: "10.77.77.23".parse().unwrap(),
+                ports: 4096..=8192,
+            }),
+            external_ips: None,
+        },
+        ipv6: Ipv6Cfg {
+            vpc_subnet: "fd00::/64".parse().unwrap(),
+            private_ip: "fd00::5".parse().unwrap(),
+            gateway_ip: "fd00::1".parse().unwrap(),
+            snat_cfg: Some(SNat6Cfg {
+                public_ip: "2001:db8::1".parse().unwrap(),
+                ports: 1025..=4096,
+            }),
+            external_ips: None,
+        },
+    };
     VpcCfg {
-        private_ip: "172.30.0.6".parse().unwrap(),
-        private_mac: MacAddr::from([0xA8, 0x40, 0x25, 0xF0, 0x00, 0x66]),
-        vpc_subnet: "172.30.0.0/22".parse().unwrap(),
-        snat: Some(SNatCfg {
-            // NOTE: This is not a routable IP, but remember that a
-            // "public IP" for an Oxide guest could either be a
-            // public, routable IP or simply an IP on their wider LAN
-            // which the oxide Rack is simply a part of.
-            public_ip: "10.77.77.23".parse().unwrap(),
-            ports: 4097..=8192,
-            phys_gw_mac: MacAddr::from([0x78, 0x23, 0xae, 0x5d, 0x4f, 0x0d]),
-        }),
-        external_ips_v4: None,
-        gw_mac: MacAddr::from([0xA8, 0x40, 0x25, 0xFF, 0x77, 0x77]),
-        gw_ip: "172.30.0.1".parse().unwrap(),
+        ip_cfg,
+        private_mac: MacAddr::from([0xA8, 0x40, 0x25, 0xFA, 0xFA, 0x37]),
```

Comment:
I don't think you meant to give guest2 the same mac as guest1.

Reply:
I did not, no. That looks like a copy-paste error, thanks for catching!